lib/katello/tasks/upgrades/3.0/delete_docker_v1_content.rake

```diff
@@ -0,0 +1,18 @@
+namespace :katello do
+  task :delete_docker_v1_content do
```

Comment:
These upgrade tasks have tended to follow a naming scheme as well (see - https://github.com/Katello/katello-installer/blob/master/hooks/post/30-upgrade.rb). Also, a reminder, given that link you'll have to add these to the installer to be run (if you already were accounting for that ignore me).